lib/src/observe/coap_observe_relation.dart

```diff
@@ -14,15 +14,16 @@ class CoapObserveRelation {
   /// The observing endpoint
   /// The observed resource
   /// The exchange that tries to establish the observe relation
-  CoapObserveRelation(CoapObservingEndpoint endpoint, CoapIResource resource,
-      CoapExchange exchange) {
+  CoapObserveRelation(this.cfg, CoapObservingEndpoint endpoint,
+      CoapIResource resource, CoapExchange exchange) {
     _endpoint = endpoint;
     _resource = resource;
     _exchange = exchange;
     _key = '$source#${exchange.request!.tokenString}';
   }
 
-  final CoapILogger? _log = CoapLogManager().logger;
+  DefaultCoapConfig cfg;
```

Comment:
`cfg` could be renamed to `config`

Reply:
Fixed: https://github.com/shamblett/coap/pull/76/commits/ad10f99673d9303bd2709d6c05a78c065909d09b